src/main/kotlin/app/batch/S3DirectoryReader.kt

```diff
@@ -27,17 +26,16 @@ class S3DirectoryReader(private val s3Client: AmazonS3, private val s3Utils: S3U
     @Value("\${s3.bucket}") //where the HTME exports and the Sender picks up from
     lateinit var s3BucketName: String
 
-
     override fun read(): EncryptedStream? {
         val iterator = getS3ObjectSummariesIterator(s3Client, s3BucketName)
         return if (iterator.hasNext()) {
             val next = iterator.next()
             val inputStream = getS3ObjectInputStream(next, s3Client, s3BucketName)
             val metadata = getS3ObjectMetadata(next, s3Client, s3BucketName)
-            logger.info("Returning s3 object for '${next.key}' with metadata '$metadata'")
+            logger.info("Returning s3 object from directory", "file_name" to next.key,
+                    "metadata" to metadata.map { it.key to it.value }.toTypedArray().toString())
```

Comment:
This log line is a little odd - we are mapping the key value pairs to object `Pair<>`s but then passing them all to a `metadata` log key? I would suggest we log them as a single string or split them all up into their k:v pairs